addons/mail/static/src/components/call_settings_menu/call_settings_menu.xml

```diff
@@ -2,9 +2,9 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.CallSettingsMenu" owl="1">
-        <div class="o_CallSettingsMenu" t-attf-class="{{ className }}" t-ref="root">
-            <div class="o_CallSettingsMenu_category d-flex flex-column overflow-auto">
-                <div class="o_CallSettingsMenu_categoryTitle fw-bolder text-700 text-truncate text-uppercase">Voice Settings</div>
+        <div class="o_CallSettingsMenu user-select-none" t-attf-class="{{ className }}" t-ref="root">
+            <div class="o_CallSettingsMenu_category d-flex flex-column pt-2 px-3 overflow-auto">
```

Comment:
I don't get why change `py-2 px-3` on `o_CallSettingsMenu` by `pt-2 px-3` or `pb-2 px-3` on `o_CallSettingsMenu_category`.

Visually it's the same, but it's conditionally `t` or `b`, which makes it harder to maintain because now it's dependent on whether it's the first or last category (and what if another category in the middle?). So it's more painful to maintain.

The purpose is just padding around the call setting menu, so I would keep it there. Do you see some problems with this approach?

Reply:
Since `CallSettingsMenu_category` has `overflow-auto`, it hides part of the outline when the checkbox is focused. That's why I wanted to put the padding on `o_CallSettingsMenu_category`.
But you're right, this is not a good approach.

I updated the branch and put a `py-2` on `o_CallSettingsMenu` and left the `px-3` on each `o_CallSettingsMenu_category`. It's easier to maintain. What do you think ?

![demo](https://user-images.githubusercontent.com/80679690/189894578-d40dff3f-4d29-4e50-8952-ba6b7da56b2a.jpg)

Reply:
LGTM, thanks!